app/config/environments.php

```diff
@@ -82,13 +95,44 @@
         'image' => 'appwrite/env-deno-1.6:1.0.0',
         'build' => '/usr/src/code/docker/environments/deno-1.6',
         'logo' => 'deno.png',
+        'supports' => [System::X86, System::PPC, System::ARM],
     ],
-    // 'dart-2.8' => [
-    //     'name' => 'Dart',
-    //     'version' => '2.8',
-    //     'base' => 'google/dart:2.8',
-    //     'image' => 'appwrite/env-dart:2.8',
-    //     'build' => '/usr/src/code/docker/environments/dart-2.8',
-    //     'logo' => 'dart.png',
-    // ],
-];
\ No newline at end of file
+    'dart-2.10' => [
+        'name' => 'Dart',
+        'version' => '2.10',
+        'base' => 'google/dart:2.10',
+        'image' => 'appwrite/env-dart-2.10:1.0.0',
+        'build' => '/usr/src/code/docker/environments/dart-2.10',
+        'logo' => 'dart.png',
+        'supports' => [System::X86],
+    ],
+    'dotnet-3.1' => [
```

Comment:
Why 5.1 and 3.1? anything interesting in between? we should figure how we're going to decide supported versions. For now I was trying to add all latest minor versions for each lang.

Reply:
5.0 is the latest version and 3.1 is the latest LTS release 👍🏻 
https://dotnet.microsoft.com/download